src/librustc/session/mod.rs

```diff
@@ -1078,8 +1078,7 @@ fn default_emitter(
 
 pub enum DiagnosticOutput {
     Default,
-    Raw(Box<dyn Write + Send>),
-    Emitter(Box<dyn Emitter + Send + sync::Send>)
```

Comment:
Why is this variant removed. It could be useful for custom drivers to pretty print diagnostics in their own way.

Reply:
(I replied to this a few days ago but I must have forgotten to submit it. Sorry.)

It's dead code, with no tests, no comments explaining how or why it is there. It made understanding this code harder. So I removed it. If somebody finds a concrete use for it later they can add it back in.